contracts/src/distributors/BoostedLiquidityDistributor.sol

```diff
@@ -278,19 +280,23 @@ contract BoostedLiquidityDistributor is Initializable, Ownable {
     /*///////////////////////////////////////////////////////////////
                        INTERNAL DISTRIBUTION LOGIC
     //////////////////////////////////////////////////////////////*/
-    
+
     /// @dev Update user balance & total balance after boosts.
     /// @param account The LP token depositor whose balance is being updated.
     /// @param _userDeposit LP tokens deposited by the user to use as base balance.
     /// @param _totalDeposit Total LP tokens deposited in the contract.
-    /// @dev If the boostToken contract hasn't been verified before this could lead to a reentrancy attack
-    function _updateBalances(address account, uint256 _userDeposit, uint256 _totalDeposit) internal virtual {
+    /// @dev If the boostToken contract hasn't been verified before this could lead to a reentrancy attack.
+    function _updateBalances(
+        address account,
+        uint256 _userDeposit,
+        uint256 _totalDeposit
+    ) internal virtual {
         uint256 userPower = boostToken.balanceOf(account);
         uint256 totalPower = boostToken.totalSupply();
 
         // Calculate user balance after boost
         // min((userDeposit * 0.4) + (totalDeposit * userVotingPower / totalVotingPower * 0.6), (userDeposit * 0.4))
```

Comment:
`userVotingPower` and `totalVotingPower` need to be renamed.

Reply:
@aahna-ashina can you elaborate? I think userVeNation and totalVeNation is more descriptive, I don't know if that's what you mean

Reply:
@luisivan Sorry, I just meant that the comment (documentation) is out of sync with the code, which might be a bit confusing for people reading the comments. Just a minor detail.

I'm assuming `userVotingPower` refers to `userPower` in the code. And that `totalVotingPower` refers to `totalPower`.